openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -1,3 +1,69 @@
 """
 Views for `admin_settings` app
 """
+from django.contrib.auth.models import Group, User
+from django.db.models import OuterRef, Subquery
+from django.views.generic import ListView
+
+from openedx.adg.admin_dashboard.admin_settings.constants import ADMIN_SETTINGS_PAGE_SIZE
+from openedx.adg.admin_dashboard.admin_settings.models import Invitation
+from openedx.adg.admin_dashboard.views import EnsureAdminUserAuthenticatedMixin
+from openedx.adg.lms.applications.constants import ADG_ADMIN_GROUP_NAME
+from openedx.adg.lms.applications.models import BusinessLine
+
+
+class AdminAccountListView(EnsureAdminUserAuthenticatedMixin, ListView):
+    """
+    `ListView` for active admin accounts.
+    """
+
+    queryset = None
+    template_name = 'adg/admin_dashboard/admin_settings/accounts.html'
+    http_method_names = ('get',)
+    paginate_by = ADMIN_SETTINGS_PAGE_SIZE
+    context_object_name = 'admins'
+
+    def get_queryset(self):
+        """
+        Get all the active admin accounts and prefetch admin groups.
+
+        Returns:
+            Queryset: Active admin users.
+        """
+        admin_group_names = [*BusinessLine.objects.values_list('title', flat=True), ADG_ADMIN_GROUP_NAME]
+
+        admin_users = User.objects.filter(
+            is_active=True,
+            groups__in=Group.objects.filter(
+                name__in=admin_group_names
+            )
+        ).annotate(
+            admin_group_name=Subquery(
+                Group.objects.filter(
+                    name__in=admin_group_names, user__id__in=OuterRef('id')
+                ).values('name')[:1]
+            )
```

Comment:
Let's extract this subquery into a variable and give it a meaningful name for better readability of this. Also, can't we just do `.values('name')[0]` instead of the slice like `.values('name)[:1]`

Reply:
I don't think there is any need to create a variable and with annotate its much more readable in place. And no It needs to be a slice cannot do `[0]`.